mass_editing/wizard/mass_editing_wizard.py

```diff
@@ -3,14 +3,19 @@
 
 from lxml import etree
 
-from odoo import _, api, models
+from odoo import _, api, fields, models
 
 
 class MassEditingWizard(models.TransientModel):
     _name = "mass.editing.wizard"
     _inherit = "mass.operation.wizard.mixin"
     _description = "Wizard for mass edition"
 
+    use_active_domain = fields.Boolean(
```

Comment:
This is the technical term, which might not make a lot of sense to the users. Maybe you could reword using something like 'search results'?